lua/mellifluous/highlights/plugins/fzf.lua

```diff
@@ -53,6 +53,7 @@ function M.set(hl, colors)
     })
     hl.set("FzfLuaFzfMarker", { fg = colors.ui_orange })
     hl.set("FzfLuaFzfPrompt", { fg = colors.fg4 })
+    hl.set("FzfLuaLivePrompt", { fg = hl.get("SpecialChar").fg })
```

Comment:
This could also be turned into a link

Reply:
You understood correctly. I first typed the text to grep (regex matching), then pressed Ctrl-G to fuzzy search across the results.

I'm amending my commit to make the live prompt red. Thanks for the feedback!

Reply:
Preview (without any local fzf customization in my terminal):

<img width="883" height="392" alt="image" src="https://github.com/user-attachments/assets/87f7cb87-3b44-405b-970e-ff30302adf49" />